src/shared/hooks/useTimeAgo.test.ts

```diff
@@ -0,0 +1,30 @@
+import { add } from 'date-fns';
+import { renderHook } from '@testing-library/react';
+
+import { useTimeAgo } from './useTimeAgo';
+
+jest.mock('react-i18next', () => ({
```

Comment:
per my understanding, it's better to move shared logic for test in src/__mocks__ to use it for all the tests
```
const reactI18Next = {
  useTranslation: () => ({
    i18n: {
      changeLanguage: () => new Promise(() => {}),
      language: 'en',
    },
  }),
  initReactI18next: {
    type: '3rdParty',
    init: () => {},
  },
};
```